### PR TITLE
Fixes traitor reputation being calculated and displayed incorrectly at round end

### PR DESCRIPTION
**Pull request by VioletN**
## About The Pull Request

This pull request fixes two minor issues with how traitor reputation was being calculated and displayed at the end of rounds, both related to reputation points not being properly divided.

1. At the end of a round, the objectives each traitor completed are displayed in the round-end report, along with the reputation points each objective is worth (the progression_reward variable). However, this variable was being divided by **600** instead of **60**, meaning that, for example, if an objective was worth 45 reputation points, it would be displayed as being worth 4.5 instead.

2. The objectives for "become a living legend" and "complete objectives" both attempt to calculate the reputation points a traitor earned by checking if a variable (respectively, progression_points and total_points, which is calculated by adding up the reward value of each completed objective) is higher than the reputation requirement set when the objective is created. The problem is that this requirement variable is divided by 60 when it's created, while the number it's being compared to is not divided at all. For reference, if your traitor uplink says you have 5 reputation points, that means you have 300 progression_points. So, if your "living legend" objective said you needed to get 300 reputation points, you would only actually need to get 5 for it to be considered successful.

## Why It's Good For The Game

This makes the primary traitor objectives actually require what they say they require, and prevents you from becoming a "living legend" by just breathing for 10 minutes. 

## Changelog

:cl:
fix: Fixed the two primary traitor objectives calculating reputation incorrectly at round end
fix: Fixed the reputation value of objectives having a misplaced decimal point in the round end report
/:cl:

### CODE REVIEW

**Comment by dalmationer** [comment]
good fix!!
We've also been talking about the idea of making 'become a living legend' include escaping alive, as you need to be alive to be living.

File: code/modules/antagonists/traitor/datum_traitor.dm [hunk 1]
```diff
@@ -167,7 +167,7 @@
 		return FALSE
 	if(!traitor.uplink_handler)
 		return FALSE
-	if(traitor.uplink_handler.progression_points < required_total_progression_points)
+	if(traitor.uplink_handler.progression_points / 60 < required_total_progression_points)
```
Comment (Mothblocks) [inline code/modules/antagonists/traitor/datum_traitor.dm:170]: I don't understand where this number comes from

And considering it comes up in 3 places, and was initially wrong the first, you should make this a define

Comment (Mothblocks) [reply code/modules/antagonists/traitor/datum_traitor.dm:170]: 1 MINUTE is 600, not 60. Something like `PROGRESSION_POINT_FACTOR` or similar, with this documentation would be good.

File: code/modules/antagonists/traitor/datum_traitor.dm [hunk 2]
```diff
@@ -1,3 +1,6 @@
+///Divide uplink_handler.progression_points by this to get the number of reputation points, which is displayed in-game. (e.g. 300 progression_points is 5 reputation points)
+#define PROGRESSION_POINT_FACTOR 60
```
Comment (Mothblocks) [inline code/modules/antagonists/traitor/datum_traitor.dm:2]: `#undef` at the end too

**Comment by Mothblocks** [comment]
Need a trailing newline

File: code/modules/antagonists/traitor/datum_traitor.dm [hunk 3]
```diff
@@ -333,3 +336,5 @@
 	sword.worn_icon_state = "e_sword_on_red"
 
 	H.update_inv_hands()
+
+#undef PROGRESSION_POINT_FACTOR
```
Comment (Mothblocks) [inline code/modules/antagonists/traitor/datum_traitor.dm:340]:
```suggestion
#undef PROGRESSION_POINT_FACTOR

```

**Comment by Mothblocks** [comment]
I wonder if that worked

**Comment by Mothblocks** [comment]
Your profile picture is cute by the way

**Pull request merged**